services/cards-consultation/src/main/java/org/opfab/cards/consultation/services/CardRoutingUtilities.java

```diff
@@ -171,6 +171,13 @@ public static boolean checkIfUserMustReceiveTheCard(CurrentUserWithPerimeters cu
             return true;
         }
 
+        if (checkInCaseOfCardSentByTheUserHimself(publisher, publisherType,
```

Comment:
Add unit test in CardRoutingUtilitiesShould

Reply:
Done.